grade-scripts/shared/list-manager.js

```diff
@@ -44,7 +44,8 @@ let getSymlinkedLocalList = (fileName) => {
     try {
         list = fs.readFileSync(path, { encoding: 'utf8' })
     } catch (e) {
-        throw new Error(`couldn't find and parse list ${fileName}, tried looking in: ${path}`)
+        console.warn(`couldn't find and parse list ${fileName}, tried looking in: ${path}`)
```

Comment:
We don't need these files for testing the new algo so I changed it to warn instead of error.